src/electron/blocker.ts

```diff
@@ -0,0 +1,36 @@
+/*!
+ * Copyright (c) 2017-2019 Cliqz GmbH. All rights reserved.
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ */
+
+// Type definition
+import * as electron from 'electron';
```

Comment:
I assume that this is the full `electron` package. Ideally we would find a way which does not require bundling all of it (which might fail anyway). I could not find separate type definitions for electron though..